stubs/Pillow/PIL/ImageQt.pyi

```diff
@@ -1,18 +1,21 @@
-from typing import Any
+from typing import Any, Literal
```

Comment:
```suggestion
from typing import Any
from typing_extensions import Literal
```

Reply:
Yes, thanks. Had fixed this but not pushed, so I'll fix my branch for this commit :P